src/content/config.ts

```diff
@@ -91,8 +91,9 @@ export const collections = {
 			description: z.string(),
 			publishDate: z
 				.string()
+				.transform((val) => new Date(val))
 				.or(z.date())
-				.transform((val) => new Date(val)),
+				.or(z.null()),
```

Comment:
Probably worth leaving this to a separate PR if we need a draft mode for posts, with the page still being built by `src/pages/blog/[slug].astro` it'll end up in the production sitemap even though it's hidden on the blog index page

Reply:
I actually just updated it to exclude it from the sitemap haha